tests/e2e/distribution/grpc_query_suite.go

```diff
@@ -1,4 +1,4 @@
-package testutil
+package distribution
```

Comment:
Personally, I am confused. Given that it uses depinject, isn't it more an integration test than an e2e test?
cc @kocubinski